common/lib/dependabot/shared_helpers.rb

```diff
@@ -83,13 +83,20 @@ def self.escape_command(command)
       Shellwords.join(command_parts)
     end
 
+    # rubocop:disable Metrics/MethodLength
     def self.run_helper_subprocess(command:, function:, args:, env: nil,
                                    stderr_to_stdout: false,
                                    allow_unsafe_shell_command: false)
       start = Time.now
       stdin_data = JSON.dump(function: function, args: args)
       cmd = allow_unsafe_shell_command ? command : escape_command(command)
       env_cmd = [env, cmd].compact
+      if ENV["DEBUG_FUNCTION"] == function
+        escaped_stdin_data = stdin_data.gsub("\"", "\\\"")
+        puts "$ cd #{Dir.pwd} && echo \"#{escaped_stdin_data}\" | #{env_cmd.join(' ')}"
+        # Pause execution so we can run helpers inside the temporary directory
+        byebug # rubocop:disable Lint/Debugger
+      end
```

Comment:
This is a bit of an eye-sore but seems useful to add debugging practices in code and keep iterating towards something better over time